index.html

```diff
@@ -37,7 +37,158 @@ <h1 >I’m Esmatullah Glad to see you!</h1>
+                    </div>
+                    <div class="project-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <button class="porject-tect-btn">html</button>
+                        <button class="porject-tect-btn">css</button>
+                        <button class="porject-tect-btn">javaScript</button>
+                    </div>
+                    <button class="regular-btn">See Project</button> 
+                </div>
+            </div>
+        </section>
+
+        <section class="primary-font about-me">
+            <div class="">
+                <h1 class="about-title">About Myself</h1>
+                <p class="about-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <button class="regular-btn">Get my resume</button> 
+                <div class="social-media-link">
+                    <ul class="contact-list">
+                        <li><a href=""><img src="icons/twitter.svg" alt="twitter icon"></a></li>
+                        <li><a href=""><img src="icons/linked.svg" alt="linkedin icon"></a></li>
+                        <li><a href=""><img src="icons/Vector1.svg" alt="social media icon"></a></li>
+                        <li><a href=""><img src="icons/github.svg" alt="github icon"></a></li>
+                        <li><a href=""><img src="icons/Vector2.svg" alt="social media icon"></a></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="skills">
+                <div>
+                    <div class="skills-title">
+                        <p>Languages</p>    
+                        <button><i class="fa-solid fa-angle-down fa-2x"></i></button>
+                    </div>
+                    <ul class="programming-lang">
+                        <li>
+                            <img src="images/javascrip.svg" alt="">
+                            <p>javaScript</p>
+                        </li>
+                        <li>
+                            <img src="images/html.svg" alt="">
+                            <p>HTML</p>
+                        </li>
+                        <li>
+                            <img src="images/css.svg" alt="">
+                            <p>CSS</p>
+                        </li>
+                    </ul>
+                </div>
+                <div class="skills-title">
+                    <p>Frameworks</p>
+                    <button><i class="fa-solid fa-angle-right fa-2x"></i></button>
+                </div>
+                <hr>
+                <div class="skills-title">
+                    <p>Skills</p>
+                    <button><i class="fa-solid fa-angle-right fa-2x"></i></button>
+                </div>
```

Comment:
- [ ] Same for technology categories ("Languages", "Frameworks" and "Skills") it will be better if we use a `ul` for their container and `li` tag for each of them, so we can follow HTML & CSS semantic.